api/api_test.go

```diff
@@ -0,0 +1,26 @@
+package api
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/planetdecred/dcrlibwallet/utils"
+)
+
+func TestGetBestBlock(t *testing.T) {
+	chainParams, err := utils.ChainParams("mainnet")
+	if err != nil {
+		log.Error("Error creating chain params.")
+		return
+	}
+
+	service := NewService(chainParams)
+	height := service.GetBestBlock()
+	if reflect.TypeOf(height).Kind() != reflect.Int32 {
+		t.Errorf("api.GetBestBlock returned %v; want unit32", reflect.TypeOf(height))
+	}
```

Comment:
seems like this check will never fail, unless Golang devs introduce some bug in Go compiler or `reflect` package, so its kinda pointless for us to do here